### DISCUSSION TITLE
ChipFrame element issues

### DISCUSSION THREAD

**Discussion by Qvadi** [iqm-finland/KQCircuits#61] [Q&A] [answered]
Hi,
The ChipFrame element is supposed to be a PCell. But I cannot see the PCell properties of this element. 
![image](https://github.com/iqm-finland/KQCircuits/assets/97676306/1f658ee5-7d31-49e7-918f-af75a8036bc6)

I installed KQcircuits by cloning it from github (not as a package in salt).

**Comment by qpavsmi** [comment, marked as answer]
Unfortunately it can get tricky to select the PCell as KLayout seem to constantly open the properties of the box that ChipFrame consists of. The methods that work for me is by double-clicking exactly on the right-most or top-most edge of the ChipFrame. A more consistent way of browsing the PCell parameters is to drag a selection around the ChipFrame then pressing `Q` key to view PCell parameters. If you see the `Element Library.ChipFrame` text in the selection then you know you have selected the PCell

![image](https://github.com/iqm-finland/KQCircuits/assets/109152229/093f74e0-3277-4226-bf6f-1289743a8304)

**Reply by Qvadi** [reply]
Thanks! your solution works.

**Comment by jheinsoo** [comment]
I agree with Pavel about the source of the problem. My workaround is to disable selection of Box objects as I rarely need it.

![image](https://github.com/iqm-finland/KQCircuits/assets/73839748/fff99641-f919-4ee9-ba4c-2b6ed3f482df)
